packages/design-system/.storybook/docs/tokensDocHelpers/components/DefinitionList/DefinitionListTypography.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { Token } from '../../../../../src/tokens/types';
+import { TokensProps } from '../../TokensTypes';
+import { StackVertical } from '../../../../../src';
+import DefinitionListItemTypography from './DefinitionListItem/DefinitionListItemTypography';
+
+import styles from './DefinitionList.module.scss';
+
+const DefinitionListTypography = ({ tokens }: TokensProps) => {
+	const filteredTokens = {
+		Headings: tokens.filter((token: Token) => token.name.includes('Heading')),
+		Paragraphs: tokens.filter((token: Token) => token.name.includes('Paragraph')),
+		Data: tokens.filter((token: Token) => token.name.includes('Data')),
+	};
+
+	return (
+		<StackVertical gap="L">
+			{Object.values(filteredTokens).map((entries: Token[], index) => (
+				<StackVertical gap="M" key={index} as="dl">
+					<dt className={styles.title}>{Object.keys(filteredTokens)[index]}</dt>
+					<StackVertical as="dl" gap="S">
```

Comment:
`dl` can't be a child of `dl`
I think it's the previious one that should not be a `dl` and so the following `dt` should not be a `dt`